clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -319,10 +319,9 @@ Changes in existing checks
   diagnostic.
 
 - Improved :doc:`readability-implicit-bool-conversion
-  <clang-tidy/checks/readability/implicit-bool-conversion>` check
-  by adding the option `UseUpperCaseLiteralSuffix` to select the
-  case of the literal suffix in fixes and fixing false positive for implicit
-  conversion of comparison result in C23.
+  <clang-tidy/checks/readability/implicit-bool-conversion>` check by adding the
+   options: `UseUpperCaseLiteralSuffix`,`CheckConversionsToBool`,
+   `CheckConversionsFromBool`.
```

Comment:
Please re-add what was already there to end up with something like: 

```
- Improved :doc:`readability-implicit-bool-conversion
  <clang-tidy/checks/readability/implicit-bool-conversion>` check
  by adding the option `UseUpperCaseLiteralSuffix` to select the
  case of the literal suffix in fixes and fixing false positive for implicit
  conversion of comparison result in C23, and by adding the
  options `CheckConversionsToBool` and
  `CheckConversionsFromBool` to configure which conversions 
  involving ``bool`` are allowed.
```

Reply:
Done Thanks.